### PR TITLE
typespec renaming

### PR DESCRIPTION
**Pull request by raych1**
rename cadl to typespec;
update maven binary location.

### CODE REVIEW

File: tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { DockerRunningModel } from './DockerRunningModel';
 export class DockerContext {
     mode: DockerRunningModel;
     readmeMdPath?: string;
-    cadlProjectFolderPath?: string;
+    typeSpecProjectFolderPath?: string;
```
Comment (maririos) [inline tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts:13]:
```suggestion
    typespecProjectFolderPath?: string;
```

Comment (maririos) [reply tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts:13]: Same in other places where this property is used

File: tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/GenerateAndBuildTask.ts [hunk 2]
```diff
@@ -34,12 +34,12 @@ export class GenerateAndBuildTask implements SDKGenerationTaskBase {
         const generateAndBuildOptions = generateAndBuildTask as GenerateAndBuildOptions;
         const runOptions = generateAndBuildOptions.generateAndBuildScript;
         const readmeMdAbsolutePath = !!this.context.readmeMdPath? path.join(this.context.specRepo.repoPath, this.context.readmeMdPath) : '';
-        const cadlProjectFolderAbsolutPath = !!this.context.cadlProjectFolderPath?
-            path.join(this.context.specRepo.repoPath, this.context.cadlProjectFolderPath) : '';
+        const typeSpecProjectFolderAbsolutPath = !!this.context.typeSpecProjectFolderPath?
```
Comment (maririos) [inline tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/GenerateAndBuildTask.ts:37]:
```suggestion
        const typespecProjectFolderAbsolutPath = !!this.context.typeSpecProjectFolderPath?
```

File: tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts (hunk 1 above)
Comment (maririos) [reply tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts:13]: We want to make sure we use `TypeSpec` or `typespec` and not `typeSpec`

File: tools/sdk-testgen/packages/autorest.testmodeler/README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ https://github.com/Azure/azure-rest-api-specs
 ## How to Generate Test Model
 
 ```
-autorest --version=3.7.3 --use=<test model extension> --output-folder=<RP package path> --testmodeler.export-codemodel --clear-output-folder=false --debug <RP config md file path>
+autorest --version=3.7.3 --use=<test model extension> --output-folder=<RP package path> --testmodeler.export-codemodel --clear-output-folder=false --debug <RP config md file path> --testmodeler.debugger
```
Comment (maririos) [inline tools/sdk-testgen/packages/autorest.testmodeler/README.md:10]: was the debugger added during testing, or we intentionally want it there

Comment (raych1) [reply tools/sdk-testgen/packages/autorest.testmodeler/README.md]: Removed it.

**Review by benbp — APPROVED** [review]

**Pull request merged**